openmc/universe.py

```diff
@@ -61,7 +63,9 @@ def __init__(self, universe_id=None, name=''):
         # Keys     - Cell IDs
         # Values - Offsets
         self._cell_offsets = OrderedDict()
-        self._num_regions = 0
+
+        if cells is not None:
```

Comment:
I suppose I could see `Universe` turning into a list-like class akin to `Tallies`, `Plots`, etc. At least one of the participants in the workshop was attempting to do this before I advised her that `Universe` didn't support the list-like notation. Although certainly not necessary, it does seem like a reasonable extension.